README.md

```diff
@@ -1,25 +1,68 @@
 # Darvaza Extra
 
-`darvaza.org/x` hosts specialized packages that only depend
-on the standard library, `darvaza.org/core` and the
-`darvaza.org/slog.Logger` interface.
+[![Go Reference][godoc-badge]][godoc]
+[![Go Report Card][goreport-badge]][goreport]
+
+`darvaza.org/x` hosts mid complexity packages with
+no big dependencies or assumptions.
+
+[godoc]: https://pkg.go.dev/darvaza.org/x
+[godoc-badge]: https://pkg.go.dev/badge/darvaza.org/x.svg
+[goreport]: https://goreportcard.com/report/darvaza.org/x
+[goreport-badge]: https://goreportcard.com/badge/darvaza.org/x
+
+[darvaza-cache]: https://pkg.go.dev/darvaza.org/cache
+[darvaza-core]: https://pkg.go.dev/darvaza.org/core
+[darvaza-penne]: https://pkg.go.dev/darvaza.org/penne
+[darvaza-resolver]: https://pkg.go.dev/darvaza.org/resolver
+[darvaza-slog]: https://pkg.go.dev/darvaza.org/slog
+[darvaza-sidecar]: https://pkg.go.dev/darvaza.org/sidecar
+[darvaza-simplelru]: https://pkg.go.dev/darvaza.org/cache/x/simplelru
+[darvaza-x-config]: https://pkg.go.dev/darvaza.org/x/config
+[darvaza-x-tls]: https://pkg.go.dev/darvaza.org/x/tls
+[darvaza-x-web]: https://pkg.go.dev/darvaza.org/x/web
+
+## Dependencies
+
+The _Darvaza Extra_ modules are built on top of a handful
+low(ish) level packages additionally to the Go Standard Library.
+
+* Our _core_ package,
+  [`darvaza.org/core`][darvaza-core].
+* Our _structured logger_ interface,
+  [`darvaza.org/slog`][darvaza-slog],
+  allowing users to hook their favourite logger.
+* And our thin and simple _LRU_ for local in-memory caching,
+  [`darvaza.org/cache/x/simplelru`][darvaza-simplelru].
 
 ## Packages
 
-### darvaza.org/x/config
+### Config
 
-[darvaza.org/x/config](https://daravaza.org/x/config) provides helpers
+[`darvaza.org/x/config`][darvaza-x-config] provides helpers
 for reading config files.
```

Comment:
I would say `dealing`, not `reading` because afair the `Config` also writes, dumps configs